joint_limits:
  right_arm_joint1:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 1.57
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0
    min_effort: 0
  right_arm_joint2:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 1.57
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0
    min_effort: 0
  right_arm_joint3:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 1.57
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0
    min_effort: 0
  right_arm_joint4:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 1.57
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0
    min_effort: 0
  right_arm_joint5:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 1.57
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0
    min_effort: 0
  right_arm_joint6:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 1.57
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0
    min_effort: 0
  right_arm_joint7:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
    has_velocity_limits: true
    max_velocity: 1.57
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: false
    max_jerk: 0
    has_effort_limits: false
    max_effort: 0
    min_effort: 0
